packages/api/test/functional/auth.spec.js

```diff
@@ -354,7 +354,7 @@ test('/auth/reset-password fails with invalid token', async ({ client }) => {
 	// now try with an expired token
 	const expiredToken = await user.generateToken('reset-pw');
 	const expiredDate = dayjs()
-		.subtract(25, 'hour')
+		.subtract(4, 'days')
```

Comment:
Aqui seria bacana trocar pelo `ttl + 1`, assim, caso o `ttl` mude, não quebrará a função 🙂 
```suggestion
		.subtract(Config.get('app.ttl') + 1, 'days')
```